src/test/features/response/tasks/whenWillYouPayTask.ts

```diff
@@ -163,5 +181,65 @@ describe('WhenWillYouPayTask', () => {
         expect(WhenWillYouPayTask.isCompleted(responseDraft)).to.be.true
       })
     })
+
+    context('isCompleted', () => {
```

Comment:
Would be great to recuse conditions we use to test validators or mock. Something for the future.